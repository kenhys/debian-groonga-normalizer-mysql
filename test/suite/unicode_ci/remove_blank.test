register normalizers/mysql

normalize NormalizerMySQLUnicodeCI " a  b   c" REMOVE_BLANK|WITH_CHECKS
