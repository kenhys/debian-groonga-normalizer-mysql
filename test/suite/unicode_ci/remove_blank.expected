register normalizers/mysql
[[0,0.0,0.0],true]
normalize NormalizerMySQLUnicodeCI " a  b   c" REMOVE_BLANK|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"ABC","types":[],"checks":[2,3,4]}]
